packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -66,6 +102,13 @@ function processRequest(
   const tracingInfo = computeRequestTracingInfo(request, configuration)
   const indexingInfo = computeIndexingInfo(sessionManager, startClocks)
 
+  const durationDiff = request.resolveDuration
+    ? toServerDuration((request.duration - request.resolveDuration) as Duration)
+    : undefined
+  const durationPercentageDiff = durationDiff
+    ? Math.round((durationDiff / toServerDuration(request.duration)) * 100)
+    : undefined
+
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I'm not sure we need those, as we can do arithmetics with measurement facets directly in Datadog

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏if we really need those, I would do something like this:
```ts
let durationDetails
if (request.resolveDuration) {
  durationDetails = {
    resolveDuration: ...,
    durationDiff: ...,
    durationPercentageDiff: ...
  }
}
```

Reply:
> we can do arithmetics with measurement facets

This was done based on @bcaudan 's suggestion. Happy to remove it if you insist, I personally think it's fine and do plan on deleting this section in a future PR, so I do not have super strong feelings either way.

> I would do something like ...

I agree its not very nice looking. I will add `nullish coalescing` and the `if` statement to make it a little nicer. I do plan on removing this section in a future PR shortly